handlers/commands/commands.js

```diff
@@ -28,52 +28,54 @@ const adminCommands = `\
 <code>/unban</code> - Removes the user from ban list.
 <code>/user</code> - Shows user's status and warns.
 <code>/addcommand &lt;name&gt;</code> - to create a custom command.
-<code>/removecommand &lt;name&gt;</code> - to remove a custom command.
+<code>/removecommand &lt;name&gt;</code> - to remove a custom command.\n
 `;
 const userCommands = `\
 <b>Commands for everyone</b>:
 <code>/staff</code> - Shows a list of admins.
 <code>/link</code> - Show the current group's link.
 <code>/groups</code> - Show a list of groups which the bot is admin in.
-<code>/report</code> - Reports the replied-to message to admins.
+<code>/report</code> - Reports the replied-to message to admins.\n
 `;
+const role = R.prop('role');
+const name = R.prop('name');
 
 const commandReferenceHandler = async (ctx) => {
 	const customCommands = await listCommands();
 
-	const role = R.prop('role');
 	const customCommandsGrouped = R.groupBy(role, customCommands);
-	const userCustomCommands = '[everyone]\n<code>' +
+	const userCustomCommands = customCommandsGrouped.everyone
```

Comment:
Unlike in Python, empty arrays are truthy in JS; change the condition to `customCommandsGrouped.everyone.length`

Reply:
My bad. `R.groupBy` represents empty groups as undefined (which is falsy) instead of empty array, because it doesn't know all the possible groups in advance.